sdks/java/core/src/main/java/org/apache/beam/sdk/options/GcsOptions.java

```diff
@@ -89,9 +89,14 @@
    * {@link ExecutorService} is compatible with AppEngine.
    */
   public static class ExecutorServiceFactory implements DefaultValueFactory<ExecutorService> {
-    @SuppressWarnings("deprecation")  // IS_APP_ENGINE is deprecated for internal use only.
+
     @Override
     public ExecutorService create(PipelineOptions options) {
+      return create();
+    }
+
+    @SuppressWarnings("deprecation")  // IS_APP_ENGINE is deprecated for internal use only.
+    public ExecutorService create() {
```

Comment:
it's being used by `sdks/java/core/src/main/java/org/apache/beam/sdk/util/BigQueryTableInserter.java:84`